docker-startup.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -eu
+
+: "${RUN_MIGRATION:=false}"
+: "${RUN_APP:=true}"
+
+if [ -n "${CERTS_PATH:-}" ]; then
+  i=0
+  truststore_pass=changeit
+  for cert in "$CERTS_PATH"/*; do
+    [ -f "$cert" ] || continue
+    echo "Adding $cert to default truststore"
+    keytool -importcert -noprompt -cacerts -storepass "$truststore_pass" -file "$cert" -alias custom$((i++))
+  done
+fi
+
+java $JAVA_OPTS -jar *-allinone.jar waitOnDependencies *.yaml
```

Comment:
we don't have this yet, and maybe this isn't the best way to do it. Could maybe just use db status check provided by dropwizard? https://www.dropwizard.io/0.7.1/docs/manual/migrations.html

Reply:
I'll comment this out for now, but feel free to amend as you continue development. This should be enough container setup to get the Jenkins build pipeline running.

Reply:
👍